examples/resources/coralogix_dashboard/dashboard.json

```diff
@@ -1,5 +1,4 @@
 {
-  "id": "pMjswgFSqRAhgiXqyzAXq",
   "name": "OpenTelemetry Collector Dashboard",
```

Comment:
I think we should also update coralogix_dashboard/resource.tf with working and updated example. I didn't manage to create the one in one in our example so i couldn't check the hexagon widget.